devtools/server/actors/replay/module.js

```diff
@@ -1064,7 +1080,10 @@ function createProtocolFrame(frameId, frame) {
   let functionLocation;
   if (frame.type == "call") {
     functionName = getFunctionName(frame.callee);
-    functionLocation = [getFunctionLocation(frame.callee)];
+    const location = getFunctionLocation(frame.callee);
+    if (location) {
+      functionLocation = [location];
+    }
```

Comment:
This should be an assert rather than a test.  considerScript() should ensure that the only frames we report to the driver are scripted frames associated with sources we have also reported to the driver.

Reply:
Cool, I wondered but wasn't 100% sure if that was the right thing to do.